cookbook/bundles/best_practices.rst

```diff
@@ -233,8 +233,8 @@ following standardized instructions in your ``README.md`` file.
     Step 2: Enable the Bundle
     -------------------------
 
-    Then, enable the bundle by adding the following line in the `app/AppKernel.php`
-    file of your project:
+    Then, enable the bundle by adding the following line in the
+    ``app/AppKernel.php`` file of your project:
```

Comment:
Oh really, my bad. @WouterJ is right here. But then, I don't see why we should change anything here at all.